recipe/conda_build_config.yaml

```diff
@@ -24,7 +24,7 @@ old_gcc_vendor:
   - conda_cos6
   - conda_cos7
   - conda_cos7
-  - conda_cos7
+  - conda
```

Comment:
What's this change about?

Reply:
For s390x, we don't need cos7 workarounds because we didn't have them in the first place.